questdb-rs/Cargo.toml

```diff
@@ -55,6 +56,8 @@ mio = { version = "1", features = ["os-poll", "net"] }
 chrono = "0.4.31"
 tempfile = "3"
 webpki-roots = "0.26.8"
+criterion = "0.5"
+rstest = "0.25.0"
```

Comment:
I'd prefer to avoid this extra dependency. I think the tests can be refactored into functions that are called for V1 and V2 variants.